packages/frontpage/drizzle.config.ts

```diff
@@ -12,11 +12,10 @@ if (!URL || !AUTH_TOKEN) {
 }
 
 export default defineConfig({
+  dialect: "turso",
   schema: "./lib/schema.ts",
-  out: "./drizzle",
-  dialect: "sqlite",
-  driver: "turso",
```

Comment:
Can we omit the driver?

Reply:
> Drizzle Kit automatically picks available database driver from your current project based on the provided dialect

https://orm.drizzle.team/docs/drizzle-config-file#driver